### DISCUSSION TITLE
Transfer Learning

### DISCUSSION THREAD

**Discussion by DYT-Flyer** [MIC-DKFZ/nnUNet#1462] [General]
Hey all,

I'm looking to to put together a transfer learning script using nnunet - has anyone been successful in this?

**Comment by hreso110100** [comment]
Hi I tried to train model on DATASET A but I was unable to reconfigure nnunet to predict on another DATASET B with correct preprocessing, but I am curious too if it is possible

**Comment by mariakawula** [comment]
I would love to use it if you managed to write such a script. I'm trying to fine-tune a pre-trained nnUNet but have problems with loading pre-trained weights. Any tips on how to solve this issue?
